ign_rviz/include/ignition/rviz/rviz.hpp

```diff
@@ -95,6 +116,7 @@ class RViz : public QObject
 
   // Plugins
   std::shared_ptr<plugins::MessageDisplay<tf2_msgs::msg::TFMessage>> tf_plugin;
+  std::shared_ptr<plugins::MessageDisplay<sensor_msgs::msg::LaserScan>> laser_scan_plugin;
```

Comment:
You are defining here just one laser scan plugin, but we should be able to create more than one, as well as be able to remove it if we don't want to use anymore. Can you think about a data structure to keep this data?

Reply:
We can use `std::vector` of plugin shared pointers?